config/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    # Copyright 2015 WSO2 Inc. (http://wso2.org)
```

Comment:
Apart from year. Even this license format look old as well. Please check and update to new